src/test/resources/liquibase/harness/change/expectedSnapshot/redshift/addPrimaryKey.json

```diff
@@ -4,14 +4,14 @@
       "liquibase.structure.core.Table": [
         {
           "table": {
-            "name": "test_table"
+            "name": "posts"
           }
         }
       ],
       "liquibase.structure.core.PrimaryKey": [
         {
           "primaryKey": {
-            "name": "test_pk"
+            "_noMatch": true
```

Comment:
_noMatch = true is mostly for drop changetypes. It verifies that a specific object is absent, no fit here